week-4/D-methods/exercise-5.js

```diff
@@ -9,35 +9,64 @@ only if the inserted amount is greater or equal than the price of the coffee!
 */
 
 var coffeeMachine = {
```

Comment:
Try and run this code, you will see that the output is not correct. Here is what I get when I run it:

```
Expected result: 'Please take your cappuccino'. Actual result: Please take your cappuccino
Expected result: 'Please take your blackCoffee'. Actual result: Please take your cappuccino
Expected result: 'Please take your flatWhite'. Actual result: Please take your cappuccino
Expected result: 'Sorry you don't have enough money for a flatWhite'. Actual result: Please take your cappuccino
```

The actual result doesn't always match the expected one. Could you have another go at this exercise please?